x-pack/plugins/security_solution/public/timelines/components/timeline/query_tab_content/index.tsx

```diff
@@ -404,6 +429,9 @@ const mapDispatchToProps = (dispatch: Dispatch, { timelineId }: OwnProps) => ({
       })
     );
   },
+  onEventClosed: (args: ToggleExpandedEvent) => {
```

Comment:
when adding new props make sure to extend React.memo function as well https://github.com/elastic/kibana/pull/83963/files#diff-96075d373020abe4908e7cd2c3fe10793987bf0a1b77f1e096a255e9f506f584R455